runtime/khala/src/lib.rs

```diff
@@ -252,6 +270,28 @@ construct_runtime! {
 pub struct BaseCallFilter;
 impl Contains<Call> for BaseCallFilter {
     fn contains(call: &Call) -> bool {
+        if let Call::PolkadotXcm(xcm_method) = call {
+            match xcm_method {
+                pallet_xcm::Call::execute { .. }
+                | pallet_xcm::Call::teleport_assets { .. }
+                | pallet_xcm::Call::reserve_transfer_assets { .. }
+                | pallet_xcm::Call::limited_reserve_transfer_assets { .. }
+                | pallet_xcm::Call::limited_teleport_assets { .. } => {
+                    return false;
+                }
+                pallet_xcm::Call::force_xcm_version { .. }
+                | pallet_xcm::Call::force_default_xcm_version { .. }
+                | pallet_xcm::Call::force_subscribe_version_notify { .. }
+                | pallet_xcm::Call::force_unsubscribe_version_notify { .. }
+                | pallet_xcm::Call::send { .. } => {
+                    return true;
+                }
+                pallet_xcm::Call::__Ignore { .. } => {
+                    unimplemented!()
```

Comment:
if `unimplemented!()` triggered, the runtime will panic?
why not just `false`?

Reply:
Exactly, return false sounds better. Updated.